pkg/api/oidc_login_handler.go

```diff
@@ -38,6 +39,16 @@ func NewOIDCLoginPageHandler(sessionStore sessions.Store, oauthConfig *oauth2.Co
 			writeError(w, http.StatusInternalServerError, "Failed to save auth session")
 			return
 		}
-		http.Redirect(w, r, oauthConfig.AuthCodeURL(state), http.StatusTemporaryRedirect)
+		opts := make([]oauth2.AuthCodeOption, 0, len(r.URL.Query()))
+		for k, v := range r.URL.Query() {
+			if len(v) < 1 {
```

Comment:
Is this the same as:
```suggestion
			if v == "" {
```
?

Reply:
`v` in this context is an array.